apps/web/vibes/soul/sections/footer/index.tsx

```diff
@@ -29,23 +31,23 @@ interface ContactInformation {
 }
 
 interface Props {
-  logo?: string | Image
-  sections: Section[]
-  copyright?: string
-  contactInformation?: ContactInformation
-  paymentIcons?: ReactNode[]
-  socialMediaLinks?: SocialMediaLink[]
+  logo?: Streamable<string | Image | null>
+  sections: Streamable<Section[]>
+  copyright?: Streamable<string | null>
```

Comment:
Why are we adding nulls for the streamed response? Are those promises allowed to resolve to `null`?